### PR TITLE
Update documentation in Google Drive 

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
### Documentation changes 
fixes https://github.com/wso2-enterprise/choreo/issues/7279

### CODE REVIEW

File: drive/modules/listener/types.bal [hunk 1]
```diff
@@ -107,18 +107,18 @@ type StartPageTokenResponse record {
     string startPageToken;
 };
 
-# A change to a file or shared drive.
+# Represents the change of a file or shared drive.
```
Comment (Chuhaa) [inline drive/modules/listener/types.bal:110]: @SkNuwanTissera Shall we revert it back to `A change to a file or shared drive.`

Comment (SkNuwanTissera) [reply drive/modules/listener/types.bal:110]: Sure. Lets do

**Comment by RukshiW** [comment]
See more comments in https://github.com/ballerina-platform/module-ballerinax-googleapis.drive/pull/60/files

File: drive/modules/listener/Module.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+configurable string callbackURL = ?;
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshUrl = drive:REFRESH_URL;
+configurable string refreshToken = ?;
+configurable string domainVerificationFileContent = ?
+
+string fileName = "<NEW_FILE_NAME>";
+
+drive:Configuration config = {
```
Comment (abeykoon) [inline drive/modules/listener/Module.md:52]: Shall we call this `clientConfig`?

File: drive/modules/listener/Module.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    clientConfig: {
+        clientId: clientId,
+        clientSecret: clientSecret,
+        refreshUrl: refreshUrl,
+        refreshToken: refreshToken
+    }
+};
+
+listen:ListenerConfiguration configuration = {
```
Comment (abeykoon) [inline drive/modules/listener/Module.md:61]: Shall we call this `listenerConfig`?

File: drive/modules/listener/Module.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    port: 9090,
+    callbackURL: callbackURL,
+    domainVerificationFileContent : domainVerificationFileContent,
+    clientConfiguration: config
+};
+```
+### Step 3: Invoke listner triggers
```
Comment (abeykoon) [inline drive/modules/listener/Module.md:68]: Define Ballerina service with the listener

File: drive/modules/listener/Module.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+- Obtain tokens - Follow [this link](https://developers.google.com/identity/protocols/oauth2)
+
+## Quickstart
+
+To use the Google Calendar connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import listener
+Import the Google Drive Listener module to your Ballerina program as follows.
+i.e: The client connecter is imported to create a file in the drive here.
```
Comment (abeykoon) [inline drive/modules/listener/Module.md:33]: Shall we remove the above line please? We are just listening to the create event.

**Review by abeykoon — APPROVED** [review]

**Pull request merged**